### PR TITLE
Fix HTML OEMol codepath used in argos

### LINKED ISSUE #709: HTML fitness render is broken for `oemol` inputs with `align=True`
When using the fitness render the following causes issues when align = True

```python
        if self.align:
            # merge
            complex = self.protein
            for pos in pose: # <---- iterating over list of lists
                complex = combine_protein_ligand(complex, pos)

            # align complex to master structure
            complex_aligned, _ = superpose_molecule(
                self.reference_target,
                complex,
            )
```

because the oemol code path is not unpacked to a list of lists when read 

```python 
 for pose, path in zip(poses, output_paths):
            if pose:
                if isinstance(pose, oechem.OEMolBase):
                    mol = pose.CreateCopy() # <---- not list
                else:
                    mol_fact = MolFileFactory.from_file(
                        str(pose)
                    ).ligands  # in this way we allow multiple ligands per protein, e.g. for viewing fragments
                    mol = [mol.to_oemol() for mol in mol_fact] # <---- listcomp 
                    for m in mol:
                        oechem.OESuppressHydrogens(
                            m, True, True
                        )  # retain polar hydrogens and hydrogens on chiral centers
                self.poses.append(mol)

                self.output_paths.append(path)
```

### PR DESCRIPTION
**Pull request by hmacdope**
## Description
Fixes #709 

## Developers certificate of origin
- [X] I certify that this contribution is covered by the MIT license as defined in our [LICENSE](https://github.com/choderalab/asapdiscovery/blob/main/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).

### CODE REVIEW

**Comment by hmacdope** [comment]
@JenkeScheen could I grab a review here?

File: asapdiscovery-dataviz/asapdiscovery/dataviz/tests/test_html_viz.py [hunk 1]
```diff
@@ -16,15 +17,43 @@ def test_html_viz_subpockets(pose, protein, target, tmp_path):
...
+@pytest.mark.parametrize("align", [True, False])
+def test_html_viz_subpockets_align(pose, protein, align, tmp_path):
+    html_visualizer = HTMLVisualizer(
+        poses=[pose],
+        output_paths=[tmp_path / "html_viz.html"],
+        target="SARS-CoV-2-Mpro",
+        protein=protein,
+        align=align,
+    )
+    html_visualizer.write_pose_visualizations()
```
Comment (ijpulidos) [inline asapdiscovery-dataviz/asapdiscovery/dataviz/tests/test_html_viz.py:29]: Do we want to assert something here in the tests? What is the test actually testing?

**Review by ijpulidos — APPROVED** [review]
Looks good to me, I just made one non-blocking comment with the tests.

File: asapdiscovery-dataviz/asapdiscovery/dataviz/tests/test_html_viz.py (hunk 1 above)
Comment (hmacdope) [reply asapdiscovery-dataviz/asapdiscovery/dataviz/tests/test_html_viz.py:29]: Good point.

**Pull request merged**